Group15_FitnessAssignment/config/express.js

```diff
@@ -21,9 +22,10 @@ module.exports = function(app, passport){
     app.use(express.json());
     app.use(express.urlencoded({ extended: false }));
     app.use(cookieParser());
+    app.use(session({ secret: package.name, resave: false, saveUninitialized: true }))
```

Comment:
Im using package.name as the secret. Which is basically our app name. This just seems more robust than adding a string.